driver.go

```diff
@@ -548,6 +548,12 @@ func (d cephRBDVolumeDriver) Unmount(r dkvolume.Request) dkvolume.Response {
 	return dkvolume.Response{}
 }
 
+func (d cephRBDVolumeDriver) Capabilities(r dkvolume.Request) dkvolume.Response {
+    var res dkvolume.Response
+    res.Capabilities = dkvolume.Capability{Scope: "local"}
```

Comment:
If this was "global" or configurable, I could probably accept this patch. As-is, I would not want to set scope:local.